### ISSUE TITLE
About the code

### ISSUE THREAD

**Issue by liushz** [MARIO-Math-Reasoning/Super_MARIO#3]
Thanks for your great job for math reasoning!
I am just doing some research about this domain, and I wonder when will you release your code? I'm thinking of reproducing your paper by myself if you will not publish it soon (like in a week).

**Comment by lovecambi** [comment]
> Thanks for your great job for math reasoning! I am just doing some research about this domain, and I wonder when will you release your code? I'm thinking of reproducing your paper by myself if you will not publish it soon (like in a week).

Thanks. The code will be released in one or two days.

**Comment by billxbf** [comment]
I saw you've uploaded some decoding code for inference. Thanks! I wonder if you plan to release training code, like loss implementation, etc. in the future?

**Comment by lovecambi** [comment]
> I saw you've uploaded some decoding code for inference. Thanks! I wonder if you plan to release training code, like loss implementation, etc. in the future?

We just release the mcts inference code, which is the core part to construct the training data. For the loss, we just used LLM's `auto-regressive loss` + 0.1 * `value loss`, where `value loss` is simply MSE.

Due to corporate policy, we currently cannot release the training code.
